src/views/screen-too-small.cpp

```diff
@@ -32,5 +32,8 @@ namespace views{
         return false;
     }
 
+    const char* ScreenTooSmallView::getName(){
+        return name_;
+    }
```

Comment:
@giospada il pointer però sarebbe da mettere dall'altra parte D:

Reply:
Nevermind, ho visto che hai fatto così praticamente in tutta la parte del tuo codice, chissene.